python-backend/app/__init__.py

```diff
@@ -59,7 +60,7 @@ def register_routes(app):
     api.add_namespace(permits_api)
     api.add_namespace(document_api)
     api.add_namespace(document_manager_api)
-
+    api.add_namespace(users_api)
```

Comment:
This doesn't support splitting the service out into minespace API in the future.

Reply:
Users is registered to the app as one namespace and minespace is a submodule. I see your point. maybe it should be reversed. /minespace/users/ instead of users/minespace/....